.changeset/sixty-spies-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
+
+Map Lozenge Orbiter variants to Badge
```

Comment:
Can you rewrite the release notes in a way that is more telling to our users? Also there seems to be changes to Tag, but it's not documented